com.github.alexhuntley.Plots.json

```
@@ -0,0 +1,50 @@
+{
+  "app-id": "com.github.alexhuntley.Plots",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "command": "plots",
+  "finish-args": [
+    "--socket=fallback-x11",
```

For some reason, the app won't start on x11 for me when using fallback-x11 
```suggestion
    "--socket=x11",
```

---

there's probably an issue with your machine then :P do you have any output when you run it from the terminal using `flatpak run com.github.alexhuntley.Plots`?

---

Just tried it again and it worked fine, not sure why. The output before was something about not being able to connect to the display.

---

Ah, classic one. The issue comes from something else but I couldn't find the time to track what causes it exactly and I'm not able to reproduce it for a long time now. I think it's might be caused by gdm or whatever that handles the login to specific session on your system